x-pack/plugins/security_solution/public/common/mock/get_mock_theme.ts

```diff
@@ -0,0 +1,12 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { RecursivePartial } from '@elastic/eui/src/components/common';
+import { EuiTheme } from '../../../../../../src/plugins/kibana_react/common';
+
+export const getMockTheme = (partialTheme: RecursivePartial<EuiTheme>): EuiTheme =>
```

Comment:
This might be more appropriate in `x-pack/plugins/security_solution/public/common/lib/kibana/kibana_react.mock.ts`; that's where the rest of our mock helpers for `kibana_react` exist right now. At this point, it might be a good idea to open a PR against the kibana_react plugin to make these "official" mocks [as described here](https://github.com/elastic/kibana/blob/master/src/core/TESTING.md#plugin-integrations).

Reply:
updated